src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -175,16 +179,31 @@ export class CdkMenuItemTrigger implements OnDestroy {
   }
 
   /**
-   * If there are existing open menus and this menu is not open, close sibling menus and open
-   * this one.
+   * Subscribe to the mouseenter events and close any sibling menu items if this element is moused
+   * into.
    */
-  _toggleOnMouseEnter() {
-    const menuStack = this._getMenuStack();
-    const isSiblingMenuOpen = !menuStack.isEmpty() && !this.isMenuOpen();
-    if (isSiblingMenuOpen) {
-      this._closeSiblingTriggers();
-      this.openMenu();
-    }
+  private _subscribeToMouseEnter() {
+    // Closes any sibling menu items and opens the menu associated with this trigger.
+    const toggleMenus = () =>
+      this._ngZone.run(() => {
+        this._closeSiblingTriggers();
+        this.openMenu();
+      });
```

Comment:
Rather than passing a callback, we'd typically just have the menuAim API return an `Observable`. What if menu aim returned a stream that emits immediately if the aim doesn't go towards the submenu, or after the delay if the cursor ends up somewhere else, or not at all if it actually ends up in the submenu.

Reply:
We can't have an observable shared across all calls to `menuAim.toggle()` since each call is unique (unless we do what we do with MenuStack where we emit the menu being closed). Therefore we would have to create a new observable for each call which didn't seem ideal. I'm open to either approach here though.